src/NodeResultPrinting.cpp

```diff
@@ -158,7 +158,8 @@ static void PrintDiagnosticFormat(const char* title, const char* formatString, .
 
 static void PrintDiagnostic(const char* title, const char* contents)
 {
-    PrintDiagnosticFormat(title, "%s", contents);
+    if (contents != nullptr)
```

Comment:
I think the if check should be at the callsite instead.  there's several invocations to this (like env var values), that if they are null i do want to see them

Reply:
I inspected all calls and I thought that was ok. Environment variable printing is actually done through a different function (raw printf)